lib/rake/build.rake

```diff
@@ -56,19 +56,6 @@ namespace :build do
     end
   end
 
-  # TODO: (brent) - temporarily leave in a build step that just does a clean of
-  # code-studio to make sure we don't have artifacts from old builds
-  desc 'Builds code studio.'
-  task :code_studio do
-    if File.exist?(code_studio_dir)
-      Dir.chdir(code_studio_dir) do
-        HipChat.log 'Removing <b>code-studio</b>...'
-        RakeUtils.system 'rm -rf build'
-      end
-    end
-  end
-  task :'code-studio' => :code_studio
-
```

Comment:
same as comment in ci_build. I'm not sure which ends up being called as part of adhoc.